### ISSUE TITLE
No module named 'efficientdet'

### ISSUE THREAD

**Issue by fengduanqiao** [Guillem96/efficientdet-tf#9]
Where can I download 'efficient det'  model!thank you

**Comment by Guillem96** [comment]
Take a look at the examples folder. Try run the colab notebook

**Comment by fengduanqiao** [comment]
Ijust  want to train my model in local pycharms，What should I do?

**Comment by Guillem96** [comment]
The installation does not work for you?

**Comment by fengduanqiao** [comment]
Colab needs mobile phone verification, but Google can't identify my phone number and can't use colab

**Comment by fengduanqiao** [comment]
![image](https://user-images.githubusercontent.com/5849592/79557342-f9c3c400-80d4-11ea-9871-0a2cd6c0c304.png)
Collab is ready to run, but my goal is to train my model in local pycharm. What should I do?

**Comment by fengduanqiao** [comment]
Collab is ready to run, but my goal is to train my model in local pycharm. What should I do?







------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Guillem Orellana Trullols"<notifications@github.com&gt;;
发送时间:&nbsp;2020年4月17日(星期五) 下午5:29
收件人:&nbsp;"Guillem96/efficientdet-tf"<efficientdet-tf@noreply.github.com&gt;;
抄送:&nbsp;"风断桥"<751835019@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [Guillem96/efficientdet-tf] No module named 'efficientdet' (#9)





 
The installation does not work for you?

**Comment by fengduanqiao** [comment]
Iin pycharm I executed the following code:
python -m efficientdet.train --efficientdet 0 --bidirectional --no-freeze-backbone --format labelme --train-dataset test/data/pokemon --images-path test/data/pokemon --classes-names treecko,greninja,mewtwo,solgaleo,psyduck --n-classes 5 --epochs 20 --batch-s
ize 8 --w-scheduler --learning-rate 1e-2 --grad-accum-steps 2 --save-dir models/pokemon-models/

and Program error：

![111](https://user-images.githubusercontent.com/5849592/79629703-444d4b00-817e-11ea-8177-aa76bfc817b3.png)



How should I deal with this problem?

**Comment by Guillem96** [comment]
For the compound scaling, I am using typed NamedTuples. They are pretty new and some api details have changed recently. Check [here](https://docs.python.org/3/library/typing.html#typing.NamedTuple) if your python version is the correct for working with NamedTuples.

I am working with python 3.6.10 and all works perfectly.

Also, make sure you do a new git clone of master branche because I fixed an issue related to that.

**Comment by fengduanqiao** [comment]
![111](https://user-images.githubusercontent.com/5849592/79632542-913b1c80-8192-11ea-8e68-158c2c3cc43c.png)

https://github.com/Callidior/keras-applications/releases/download/efficientnet/efficientnet-b0_weights_tf_dim_ordering_tf_kernels_autoaugment_notop.h5

This file download address is invalid. Is there any other alternate address?

**Comment by fengduanqiao** [comment]
https://github.com/Callidior/keras-applications/releases/download/efficientnet/efficientnet-b0_weights_tf_dim_ordering_tf_kernels_autoaugment_notop.h5

This file download address is invalid. Is there any other alternate address?

**Comment by Guillem96** [comment]
I do not handle the efficientnet package. Talk with leras application team

**Comment by fengduanqiao** [comment]
Thank you

**Comment by fengduanqiao** [comment]
file:
efficientnet-b0_weights_tf_dim_ordering_tf_kernels_autoaugment_notop.h5

I also want to ask which directory this file will be placed in after downloading. Thank you

**Comment by Guillem96** [comment]
Probabily under .keras directory in your home folder

**Comment by fengduanqiao** [comment]
![image](https://user-images.githubusercontent.com/5849592/79635622-7e314800-81a4-11ea-9520-cc03e201ec17.png)

I have downloaded an .H5 file. How to avoid downloading again!

**Comment by Guillem96** [comment]
Again, I do not maintain this part of the code. You should talk with keras applications team.
